spec.html

```diff
@@ -32711,6 +32674,18 @@ <h1>Array.prototype.sort ( _comparefn_ )</h1>
           <p>The `sort` function is intentionally generic; it does not require that its *this* value be an Array object. Therefore, it can be transferred to other kinds of objects for use as a method.</p>
         </emu-note>
 
+        <emu-clause id="sec-objectissparse" aoid="ObjectIsSparse">
+          <h1>Runtime Semantics: ObjectIsSparse ( _obj_, _len_ )</h1>
+          <emu-alg>
+            1. Assert: _obj_ is an Object.
+            1. Assert: _len_ is a nonnegative integer.
+            1. For each integer _i_ in the range 0 &le; _i_ &lt; _len_, do
+              1. Let _elem_ be _obj_.[[GetOwnProperty]](! ToString(_i_)).
```

Comment:
```suggestion
              1. Let _elem_ be ? _obj_.[[GetOwnProperty]](! ToString(_i_)).
```
because i believe this can throw on, for example, an uninitialized module namespace object?

Reply:
A module namespace object probably doesn't have a "length" property, so it wouldn't get this far. But in general, yes, [[GetOwnProperty]] could throw.